mxnet/inference/docker/1.9/py3/Dockerfile.ec2.cpu.os_scan_allowlist.json

```diff
@@ -1,7 +1,6 @@
 {
    "cryptography":[
       {
-         "reason_to_ignore": "No fix published yet",
```

Comment:
@shantanutrip It appears that we have added lots of packages to this, but we have not updated the reason to ignore. Can you add back this reason to ignore and provide one for the packages added?

Reply:
Thanks for catching this. I have reverted this change.